### ISSUE TITLE
Debian Package: add attr to Ubuntu dependencies

### ISSUE THREAD

**Issue by marconett** [rear/rear#1839]
#### Relax-and-Recover (ReaR) Issue Template

* Relax-and-Recover 2.4-git.3020.aa7b197.master / 2018-06-21
* Ubuntu 16.04.1 LTS x86_64

Brief description of the issue:

`dpkg -i <package>`  after `make deb` fails with the following:
```
Selecting previously unselected package rear.
(Reading database ... 289777 files and directories currently installed.)
Preparing to unpack .../rear_2.4-git.3020.aa7b197.master_amd64.deb ...
Unpacking rear (2.4-git.3020.aa7b197.master) ...
dpkg: dependency problems prevent configuration of rear:
 rear depends on attr; however:
  Package attr is not installed.

dpkg: error processing package rear (--install):
 dependency problems - leaving unconfigured
Processing triggers for man-db (2.7.5-1) ...
Errors were encountered while processing:
 rear
```

Work-around: `apt-install attr`
Solution: Add package `attr` to dependencies.

**Comment by gdha** [comment]
The `control` file does contain it:
````
Depends: syslinux[!ppc64el], syslinux-common[!ppc64el], ethtool, ${shlibs:Depends}, lsb-release, iputils-ping, dosfstools, binutils, parted, openssl, gawk, attr, bc, ${misc:Depends}
````
Why wasn't it picked up?

**Comment by marconett** [comment]
I looked into the `DEBIAN/control` file in the build package and the line indeed contains attr: `Depends: syslinux, syslinux-common, ethtool, lsb-release, iputils-ping, dosfstools, binutils, parted, openssl, gawk, attr, bc`

I now did the following: `apt install attr; apt remove ethtool` to see if it would pick up ethtool as a dependency. It did not, as I now get the error `Package ethtool is not installed.` when trying to install rear.

I'm not really familiar with the building packages, so I'm not sure what else to try.
